app/javascript/components/tooltips/index.js

```diff
@@ -0,0 +1 @@
+export { Student } from './Student'
```

Comment:
I would encourage more descriptive names. What is a "student" tooltip?

Reply:
The context here is the tooltip shown for a student on the mentor's inbox/queue page.

Reply:
Some options:

- `MentoredStudent`
- `Mentoring.Student` (namespacing)

Reply:
I've gone with `MentoredStudent` for now.

Reply:
Ah. Makes more sense now. 

Student details in mentoring context, but technically could be used for anything if the URL was different?